yin_yang/__main__.py

```diff
@@ -139,5 +141,8 @@ def systray_icon_clicked(reason: QSystemTrayIcon.ActivationReason):
         logger.debug('System tray is unsupported')
 
     window = main_window_connector.MainWindow()
-    window.show()
+    if arguments.minimized:
+        window.close()
```

Comment:
Why close the window? Wouldn't it be enough to not open it, maybe even not instantiating it?

Reply:
Would this new logic be better? [082a203](https://github.com/oskarsh/Yin-Yang/pull/265/commits/082a203969b679f3b671dc88649ae669f76a186c)